view/frontend/templates/turnto-config.phtml

```diff
@@ -0,0 +1,17 @@
+<?php /** @var \TurnTo\SocialCommerce\Block\TurnToConfigInterface $block */ ?>
+<script type="text/javascript">
+    (function(window) {
+        window.turnToConfig = <?= $block->getJavaScriptConfig() ?>;
+
+        var customConfiguration = <?= $block->getCustomJavaScriptConfiguration() ?>;
+
+        // Add the keys from the user-supplied object to the existing turnToConfig object
```

Comment:
Change this to:

```
// Add the keys from the user-supplied object to the existing turnToConfig object
// Doing this merging in JS since it's not possible to handle this merge in PHP since there are JS _functions_ that need to get merged, not just JSON data
```